community/kernel/src/main/java/org/neo4j/kernel/impl/proc/LoggingService.java

```diff
@@ -0,0 +1,26 @@
+package org.neo4j.kernel.impl.proc;
+
+/**
+ * LoggingService provides a logging facility for procedures. It should only be used
+ * within a procedure call
+ */
+public interface LoggingService
```

Comment:
Is `org.neo4j.kernel.impl.proc` public API? The package name doesn't sound like it.